tests/html5skin-test.js

```diff
@@ -448,4 +448,54 @@ describe('Controller', function() {
 
   });
 
+  describe("Show player controls over ads", function() {
```

Comment:
Can we add tests to check the css of the plugins element and plugins click element depending on the value of playerControlsOverAds?